build.gradle

```diff
@@ -50,40 +54,56 @@ configurations {
 
 dependencies {
 	enigmaRuntime "net.fabricmc:stitch:0.1.2.49+"
-	enigmaRuntime "cuchaz:enigma:0.13.1.99:all"
+
+	// not transitive
```

Comment:
Why dont you just use the full version of engima? this seems horrible to maintain

Reply:
because full version ships the wrong procyon. Please!

Reply:
Fix engima then ;)